docker-compose.yml

```diff
@@ -38,15 +38,11 @@ services:
       MYSQL_DATABASE: silauth
       MYSQL_USER: silauth
       MYSQL_PASSWORD: silauth
-      COMPOSER_CACHE_DIR: /composer
       PROFILE_URL_FOR_TESTS: http://pwmanager.local/module.php/core/authenticate.php?as=ssp-hub
       ADMIN_PASS: b
       SECRET_SALT: abc123
       IDP_NAME: x
     volumes:
-      - ./composer.json:/data/composer.json
-      - ./composer.lock:/data/composer.lock
```

Comment:
Just as a heads-up on this change, removing these means that any changes made to those files will require rebuilding this docker image in order to see the effects of those changes when running tests during local development. Maybe not a big deal, but a possible "gotcha" at some point down the road. (I assume this was done to solve some problem/issue with the GitHub Action build, which is fine.) No objection, just an FYI.

Reply:
Actually, it wasn't solving a problem. I removed them after realizing that the test scripts never run composer in the `test` container.